src/corner_kick/config/webpack.base.config.js

```diff
@@ -0,0 +1,88 @@
+/**
+ * Specifies the webpack configuration for development
+ * and production. Provides support for Typescript and CSS files.
+ */
+
+const path = require('path');
+const webpack = require('webpack');
+const HtmlWebPackPlugin = require('html-webpack-plugin');
+const FriendlyErrorPlugin = require('friendly-errors-webpack-plugin');
+
+/**
+ * @description Webpack build.
+ */
+const generalWebpackBuild = {
+    stats: false,
+    // We support js, typescript, html, css, and image files.
+    // To add additional file support, add the required loader here.
+    module: {
+        rules: [
+            {
+                test: /\.tsx?$/,
+                exclude: /node_modules/,
+                use: {
+                    loader: 'ts-loader',
+                },
+            },
+            {
+                test: /\.html$/,
+                use: [
+                    {
+                        loader: 'html-loader',
+                        options: { minimize: false },
+                    },
+                ],
+            },
+            {
+                test: /\.css$/,
+                use: ['style-loader', 'css-loader'],
+            },
+            {
+                test: /\.(png|jpg|gif|svg)$/,
+                use: [
+                    {
+                        loader: 'file-loader',
+                        options: {},
+                    },
+                ],
+            },
+        ],
+    },
+    resolve: {
+        // Make sure if you add a new file support to add its extension here.
+        extensions: ['.js', '.ts', '.tsx', '.json'],
```

Comment:
what about `.css`, `.svg`, etc?

Reply:
Clarified in a comment `These are the extensions that Webpack will try to resolve (aka transpile into a .js bundle)`